port/osx/omrosdump.c

```diff
@@ -269,6 +299,34 @@ list_thread_commands(mach_port_t task_port, struct thread_command_full_64 **thre
 			goto done;
 		}
 		threads[i].cmdsize += state_int_count * sizeof(natural_t);
+#elif defined(OMR_ARCH_AARCH64) /* defined(OMR_ARCH_X86) */
```

Comment:
I think this block can be unified as well, if common names are given to constants, etc.
For example, `THREAD_STATE_COUNT` would be `x86_THREAD_STATE_COUNT` or `ARM_THREAD_STATE64_COUNT` as appropriate.

Reply:
Updated as suggested.